src/main/java/com/nike/cerberus/command/core/GenerateCertificateFilesCommandParametersDelegate.java

```diff
@@ -138,7 +138,7 @@
             },
             description = "Flag to supply when running command in an environment without TTY, such as a build job"
     )
-    private boolean tty = false;
+    private boolean isNoTty = false;
```

Comment:
nitpick: isTty is easier to reason about than is "is not false"

Reply:
at least reads quicker

Reply:
I agree, as discussed in person we need to have it negated grossly her because the flag defaults to false, and we want the flag to be optional.

The getter method is call `isTty()`